test/common/types.go

```diff
@@ -36,3 +36,14 @@ type prometheusAPIResponse struct {
 	Error     string                 `json:"error"`
 	Warnings  []string               `json:"warnings,omitempty"`
 }
+
+type Namespace struct {
+	Name                     string
+	Products                 []Product
+	PodDisruptionBudgetNames []string
+}
+
+type Product struct {
+	Name             string
+	ExpectedReplicas int32
+}
```

Comment:
In my opinion, is better to keep these types specialized per test, the code will be duplicated but it would be cleaner for me

Reply:
The problem is with all the tests being in a single package (which is right in my opinion), we will have to come up with different names for each file which isn't ideal in my opinion. This is why I think it is better to have a common type that makes sense for the tests that need it. If it starts to get unruly, maybe decide to split it back out again then but I don't think this will happen.

Reply:
I see I forget about this golang constrain, in this case there is nothing else we can do